（改）网盘直链下载助手.user.js

```diff
@@ -4922,4 +4922,4 @@
 	unsafeWindow.Panlinker = main;
 	main.init();
 })();
-// 好好学习，天天向上
\ No newline at end of file
+// 好好学习，天天向上
```

Comment:
I'm sorry but I have no idea why there is a difference in this line.

Just discard that if you like.